php/libraries/Database.class.inc

```diff
@@ -549,106 +549,6 @@ class Database
         $this->affected = $result;
     }
 
-    /**
-     * Selects data from the database
-     *
-     * Selects any amount of data from the database and stores the
-     * result into the $result array (accepted by reference - it will
-     * be overwritten so make sure it's empty or garbage before
-     * passing it in...)
-     *
-     * @param string $query  the SQL string which defines the SELECT query.
-     * @param array  $result an array into which the results of the select
-     *                       will be stored - this array gets overwritten completely
-     *
-     * @return     none
-     * @access     public
-     * @deprecated Use pselect wrappers instead
-     */
-    function select($query, &$result)
-    {
-        error_log(
-            "LORIS's Database::select() wrappers are deprecated and will be "
-            . "removed in a future version of LORIS. Please use prepared statements "
-            . " in the Database::pselect() variety instead."
-        );
-        $result = array();
-
-        if (strpos(strtoupper($query), 'SELECT') !== 0
-            && strpos(strtoupper($query), 'SHOW') !== 0
-            && strpos(strtoupper($query), 'DESCRIBE') !== 0
-        ) {
-            throw new DatabaseException(
-                "Only SELECT statements can be run with Database::select().",
-                $query
-            );
-        }
-
-        try {
-            $this->_printQuery($query);
-            $stmt = $this->_PDO->query($query);
-            if ($stmt) {
-                $result = $stmt->fetchAll(PDO::FETCH_ASSOC);
-            } else {
-                // This will be caught directly below
-                throw new Exception("Error");
-            }
-            return $result;
-        } catch(Exception $e) {
-            throw new DatabaseException("Invalid SELECT statement", $query);
-        }
-    }
-
-    /**
-     * Selects a single row from the database
-     *
-     * Selects a single row from the database and stores the
-     * result into the $result array (accepted by reference - it will
-     * be overwritten so make sure it's empty or garbage before
-     * passing it in...)
-     *
-     * *** This should not be used as it does not support prepared statements.
-     * *** It is left here for legacy reasons/instruments
-     *
-     * @param string $query  the SQL string which defines the SELECT query.
-     * @param array  $result an array into which the results of the select will
-     *                       be stored - this array gets overwritten completely
-     *
-     * @return     void
-     * @access     public
-     * @deprecated Use pselect wrappers instead
-     */
-    function selectRow($query, &$result)
```

Comment:
`docs/instruments/NDB_BVL_Instrument_mri_parameter_form.class.inc` will be sad

Reply:
resolved